build/instrumentation.build.js

```diff
@@ -9,33 +9,32 @@ import path from 'path';
 import copy from 'rollup-plugin-copy';
 import del from 'rollup-plugin-delete';
 import htmlTemplate from 'rollup-plugin-generate-html-template';
-import postcss from 'rollup-plugin-postcss';
 import sourcemaps from 'rollup-plugin-sourcemaps';
+import styles from 'rollup-plugin-styles';
 import { terser } from 'rollup-plugin-terser';
 import typescript from 'rollup-plugin-typescript2';
 import visualizer from 'rollup-plugin-visualizer';
 import vue from 'rollup-plugin-vue';
-import json from '@rollup/plugin-json';
+import polyFillsWrapper from '../node_modules/@empathyco/x-components/build-helpers/plugins/polyfills-wrapper.plugin';
 
 const jsOutputDirectory = path.join(process.cwd(), 'dist');
 
-const postCSSPlugins = [
-  autoprefixer({ grid: true }),
-  cssnano({ preset: ['default', { mergeLonghand: false }] })
-];
+const postCSSPlugins = [autoprefixer(), cssnano({ preset: ['default', { mergeLonghand: false }] })];
 
 /**
  * Creates a rollup configuration for projects that use X-Components. This configuration can be customized with the options object.
  *
- * @param {boolean} extractCSS - If true, the build will generate a `.css` and a `.js` file.
+ * @param {boolean} extractCss - If true, the build will generate a `.css` and a `.js` file.
+ * @param {string} outputCss - If `extractCss` is true, the build will generate a `.css` in this path.
```

Comment:
* @param {string} outputCss - If `outputCss` is true, the build will generate a `.css` in this path.